example/src/index.jsx

```diff
@@ -28,7 +40,8 @@ FocusStyleManager.onlyShowFocusOnTabs();
 const PANELS = {
   "Data Table": TableView,
   "Pivot Table": PivotView,
-  "Raw response": DebugView
+  "Raw response": DebugView,
+  "Vizbuilder": VizbuilderPanel
 };
```

Comment:
I'm open in the future to change this object into an Array of objects, where we can have more control of the properties and the order of the tabs in the UI.